vspace/src/main/java/edu/asu/diging/vspace/web/staff/UpdateShowUnpublishedLinksController.java

```diff
@@ -29,4 +29,17 @@ public String updateStatus(HttpServletRequest request,RedirectAttributes attribu
         attributes.addAttribute("showAlert", "true");
         return "redirect:/staff/space/{spaceId}";
     }
+    
+    @RequestMapping(value="/staff/space/{spaceId}/hideSpaceLinksFromPublishedSpace", method=RequestMethod.POST)
+    public String updateStatusForPublishedSpace(HttpServletRequest request,RedirectAttributes attributes, @PathVariable("spaceId") String spaceId,
+    		@RequestParam("hideSpaceLinksPublishedSpaceParam") Boolean hideLinksToUnpublishedSpaces) {
+	
+		ISpace space = spaceManager.getSpace(spaceId);
+		space.setHideLinksToUnpublishedSpaces(hideLinksToUnpublishedSpaces);
+		spaceManager.storeSpace(space, null, null);
+		attributes.addAttribute("alertType", "success");
+		attributes.addAttribute("message", "Hide all space links to this Space successfully updated!");
```

Comment:
Let's make the message either "All links to this space are now hidden." or "All links to this space are now shown."